vktrace/src/vktrace_common/optimization_function.cpp

```diff
@@ -0,0 +1,55 @@
+#include <stdio.h>
+#include <string.h>
+#include <ppl.h>
+#include <stdlib.h>

```

Comment:
ppl.h is not found on Linux. Any suggestions? Probably this code needs to be rewritten using C++11 STL constructs.